src/App.jsx

```diff
@@ -13,46 +15,91 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const SORT_BY_LENGTH = 'length';
+const SORT_BY_ALHPABET = 'alpha';
+const SET_REVERSE = 'reverse';
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const getSortedGoods = (arrayOfGoods, { sortValue, reverseValue }) => {
```

Comment:
```suggestion
const getSortedGoods = (listOfGoods, { sortValue, reverseValue }) => {
```
try to avoid such words as array, item, obj etc. when naming